src/components/ErrorBoundary/BaseErrorBoundary.js

```diff
@@ -40,10 +44,19 @@ class BaseErrorBoundary extends React.Component {
         BootSplash.hide({fade: true});
     }
 
+    onRefresh() {
+        this.setState({hasError: false});
+    }
+
+    onSignOut() {
+        Session.signOut();
+        this.onRefresh();
+    }
```

Comment:
Do we need to pass it as a prop to the component? Why can't we directly add it to the `GenericErrorPage`

Reply:
I misunderstood your comment here https://github.com/Expensify/App/pull/6856#discussion_r777004505. 

Get it now, will update